README.md

```diff
@@ -1 +1,134 @@
-# Hello-world
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+  <h3><b>Hello World</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [Hello_World] <a name="about-project"></a>
+The goal of this project master the use of linters, gitignore and best practices in creating a README file and making a professional pull request. No complex coding is used in this project.
+
+## 🛠 Built With <a name="built-with"></a>
+1- HTML\
+2- CSS\
+3- Linters
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Added index.html file
+- Added style.css file
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1- Web browser\
+2- Code editor\
+3- Git-smc
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this command:
+
+```sh
+  cd my-folder
+  git clone https://github.com/Wineshuga/Hello-world.git
+```
+
+### Install
+
+Install this project with:
+
+> No installation required
+
+### Usage
+
+To run the project: 
+
+Open index.html usig live server
+
+### Run tests
+
+To run tests, run the following command:
+
+> Coming soon
```

Comment:
[Optional] You can add 'npm install" in your Install section and as You don't have anything in this project to Usage or test. Please remove them. It will make your file clear to everyone.